must-gather/Dockerfile

```diff
@@ -13,21 +13,32 @@ RUN curl --location --output velero.tgz https://github.com/openshift/velero/arch
     curl --location --output restic.tgz https://github.com/openshift/restic/archive/refs/heads/${RESTIC_BRANCH}.tar.gz && \
     tar -xzvf restic.tgz && cd restic-${RESTIC_BRANCH} && \
     CGO_ENABLED=0 GOOS=${TARGETOS:-linux} GOARCH=${TARGETARCH} go build -a -mod=mod -ldflags '-extldflags "-static"' -o /restic github.com/restic/restic/cmd/restic && \
-    cd .. && rm -rf restic.tgz restic-${RESTIC_BRANCH}
+    cd .. && rm -rf restic.tgz restic-${RESTIC_BRANCH} && \
+    curl --location --output oc.tgz https://mirror.openshift.com/pub/openshift-v4/clients/ocp/stable-4.16/openshift-client-linux-${TARGETARCH}-rhel9.tar.gz && \
+    tar -xzvf oc.tgz && rm -rf oc.tgz README.md kubectl
```

Comment:
you can use:
```
tar -xzvf oc.tgz -C . oc
```

Above will extract `oc` only to the current folder path `.`, so no need to rm -rf README.md and kubectl.

Reply:
will test

Reply:
applied